File: Problema3/Fecha.cs
using System;

class Fecha
{
    private int mes;
    private int dia;
    private int anio;

    public Fecha(int mes, int dia, int anio)
    {
        this.mes = mes;
        this.dia = dia;
        this.anio = anio;
    }

    public void MostrarFecha()
    {
        Console.WriteLine(mes + "/" + dia + "/" + anio);
    }
}

class Program
{
    static void Main()
    {
        Fecha fecha1 = new Fecha(8, 30, 2026);

        Console.Write("Fecha: ");
        fecha1.MostrarFecha();
    }
}

File: Problema3/Fecha.cpp
#include <iostream>

using namespace std;

class Fecha
{
private:
    int mes;
    int dia;
    int anio;

public:
    Fecha(int mes, int dia, int anio)
    {
        this->mes = mes;
        this->dia = dia;
        this->anio = anio;
    }

    void mostrarFecha()
    {
        cout << mes << "/" << dia << "/" << anio << endl;
    }
};

int main()
{
    Fecha fecha1(8, 30, 2026);

    cout << "Fecha: ";
    fecha1.mostrarFecha();

    return 0;
}
